bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java

```diff
@@ -70,6 +70,7 @@ class ApiVersionedDir {
     // Multiple languages:
     "package_name",
     "transport",
+    "handwritten_layer",
```

Comment:
It only belongs to TypeScript so should be one line below :)

Reply:
Hm, currently only TS uses it but it could be used by any language. Do you still want to move it?